build.ps1

```diff
@@ -2,7 +2,12 @@
 param(
     [string]$DockerfilePath = "*",
     [string]$ImageBuilderCustomArgs,
-    [switch]$CleanupDocker
+    [switch]$CleanupDocker,
+
+    # Adds "--privileged" to the "docker run" command. In unknown circumstances we encounter on the
+    # RHEL build agent, the container must be run with --privileged in order to access the shared
+    # docker socket. See https://stackoverflow.com/a/36614457 for an untried potential alternative.
+    [switch]$RunPrivileged
```

Comment:
Question:  Do you think there is more value in making this generic/flexible - e.g. [string]$RunCustomArgs?